tsconfig.json

```diff
@@ -26,6 +26,7 @@
       "@components/*": ["src/components/*"],
       "@components": ["src/components/index.ts"],
       "@abi": ["src/features/abi/index.ts"]
-    }
+    },
+    "strictNullChecks": true
```

Comment:
Why this?

Reply:
By enabling strictNullChecks, you can prevent null-related runtime errors. When strictNullChecks is set to true, the TypeScript compiler enforces stricter rules regarding null and undefined values (null and undefined values are not assignable to other types, etc.)